driver/pparse.mli

```diff
@@ -32,8 +32,8 @@ val preprocess : string -> string
 val remove_preprocessed : string -> unit
 
 type 'a ast_kind =
-| Structure : Parsetree.structure ast_kind
-| Signature : Parsetree.signature ast_kind
+| Implementation : Parsetree.implementation ast_kind
+| Interface : Parsetree.interface ast_kind
```

Comment:
I would rather keep this type unchanged for compatibility and to make the change less invasive.